python/raf/testing/common.py

```diff
@@ -272,24 +272,24 @@ def wrapper(*args, **kwargs):
 
 
 def get_dist_info(verbose=False):
```

Comment:
Given that we now have both `dist_config` and `communitcator` as user interfaces, maybe `get_dist_comm_info` would be clearer.